doc/performance-investigations/0003-session-performance/baseline1Client_stats_history.csv

```diff
@@ -0,0 +1,32 @@
+"Type","Name","Timestamp","# requests","# failures","Requests/s","Requests Failed/s","Median response time","Average response time","Min response time","Max response time","Average Content Size","50%","66%","75%","80%","90%","95%","98%","99%","99.9%","99.99%","99.999","100%"
```

Comment:
The file naming is inconsistent, they should be lowercase and underscore separated. This file should be `baseline_1_client_stats_history.csv`.